src/components/map-container.js

```diff
@@ -388,12 +388,19 @@ export default function MapContainerFactory(MapPopover, MapControl) {
       }
     }
 
+    _onViewportChange = (viewState) => {
+      if (typeof this.props.onViewStateChange === 'function' && this.props.index === 0) {
```

Comment:
When in split mode:
Kepler performs the callback only when moving the left map but not the right one.
Should kepler send updates despite of which map the user is trying to move?

even in split mode this method is called only on the container users are trying to move so we shouldn't get any duplicated event

Reply:
that's true, updated